expand.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/python3
+
+import sys
+import getopt
+import tempfile
+import subprocess
+
+usage = '''Usage:expand.py [options] <output modules>
+Output Modules:
+    convolution
+    dsu
+    fenwicktree
+    lazysegtree
+    math
+    maxflow
+    mincostflow
+    modint
+    scc
+    segtree
+    string
+    twosat
+
+You can select multiple modules for <output modules>
+    e.g.)expand.py math segtree
+
+Options:
+    -a  --all       import all modules
+    -h  --help      print help
+'''
+output_header = '//https://github.com/rust-lang-ja/ac-library-rs\n'
+opt_list = ['help', 'all']
+output_list_all = ('convolution', 'dsu', 'fenwicktree', 'lazysegtree', 'math',
+                   'maxflow',  'mincostflow', 'modint', 'scc',  'segtree',
+                   'string', 'twosat',
+                   'internal_bit', 'internal_math', 'internal_queue',
+                   'internal_scc', 'internal_type_traits',)
+dependency_list = {'convolution': ('internal_bit', 'modint',),
+                   'fenwicktree': ('internal_type_traits',),
+                   'lazysegtree': ('internal_bit',),
+                   'math': ('internal_math',),
+                   'maxflow': ('internal_type_traits', 'internal_queue',),
+                   'mincostflow': ('internal_type_traits',),
+                   'modint': ('internal_math', 'internal_type_traits',),
+                   'scc': ('internal_scc',),
+                   'segtree': ('internal_bit',),
+                   'twosat': ('internal_scc',), }
+src_path = 'src/'
+
+
+def output_file(filename):
+    global src_path
+
+    res = []
+    with open(src_path+filename+'.rs', 'r') as f:
+        res.append('mod {}{{'.format(filename))
```

Comment:
```suggestion
        res.append('pub mod {} {{'.format(filename))
```

Why not add `pub`? This will help suppressing `dead-code` warnings.